homeassistant/components/intellifire/strings.json

```diff
@@ -1,12 +1,16 @@
 {
   "config": {
+    "flow_title": "Discovered Fireplace ({host})",
     "step": {
       "manual_device_entry": {
         "description": "Local Configuration",
         "data": {
           "host": "[%key:common::config_flow::data::host%]"
         }
       },
+      "dhcp_confirm": {
+        "description": "Do you want to setup {host}?"
```

Comment:
Should we show them the serial number as well?

Reply:
Why not!